lib/rubocop/cop/style/mixin_usage.rb

```diff
@@ -48,24 +48,22 @@ class MixinUsage < Cop
 
         def_node_matcher :include_statement, <<-PATTERN
           (send nil? ${:include :extend :prepend}
-            (const nil? _))
+            const)
         PATTERN
 
         def on_send(node)
           return unless (statement = include_statement(node))
-          return unless top_level_node?(node)
+          return unless implicit_style?(node)
 
           add_offense(node, message: format(MSG, statement: statement))
         end
 
         private
 
-        def top_level_node?(node)
-          if node.parent.parent.nil?
-            node.sibling_index.zero?
-          else
-            top_level_node?(node.parent)
-          end
+        def implicit_style?(node)
```

Comment:
Not sure about the name `#implicit_style?`. I don't see how it being inside a context is "implicit" and in the top level "explicit"? 🙂

Reply:
Indeed. It was named with an implicit assumption that [3 types of support](https://github.com/bbatsov/rubocop/pull/4840#issuecomment-335071288) will be done in the future 💦  I changed it to `accepted_include` proposed by [the following comment](https://github.com/bbatsov/rubocop/pull/4941/files#r147393506).